lib/pbkit/pbkit.h

```diff
@@ -67,25 +67,42 @@ DWORD   pb_wait_for_vbl(void);  //waits for VBL, returns VBL counter value as bo
 void    pb_erase_depth_stencil_buffer(int x, int y, int w, int h);
 
 void    pb_reset(void); //forces a jump to push buffer head (do it at frame start)
-int pb_finished(void);  //prepare screen swapping at VBlank (do it at frame end)
+int  pb_finished(void); //prepare screen swapping at VBlank (do it at frame end)
                 //if it returns 1 it failed (too early, just wait & retry)
                 //that means you can draw more details in your scene
 
 void pb_wait_until_gr_not_busy(void);
 DWORD pb_wait_until_tiles_not_busy(void);
 
 DWORD   *pb_begin(void);    //start a block with this (avoid more than 128 dwords per block)
-void    pb_push1to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1); //slow functions but with debug messages
-void    pb_push2to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2);
-void    pb_push3to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3);
-void    pb_push4to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3, DWORD param4);
-void    pb_push1(DWORD *p, DWORD command, DWORD param1); //slow functions but with debug messages (targets SUBCH_3D (0))
-void    pb_push2(DWORD *p, DWORD command, DWORD param1, DWORD param2);
-void    pb_push3(DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3);
-void    pb_push4(DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3, DWORD param4);
-void    pb_push4f(DWORD *p, DWORD command, float param1, float param2, float param3, float param4);
-void    pb_push_transposed_matrix(DWORD *p, DWORD command, float *m);
-void    pb_end(DWORD *pEnd);    //end a block with this (triggers the data sending to GPU)
+
+inline
+void    pb_pushto(DWORD subchannel, DWORD *p, DWORD command, DWORD nparam);
+inline
+DWORD   *pb_push1to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1); //slow functions but with debug messages
+inline
+DWORD   *pb_push2to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2);
+inline
+DWORD   *pb_push3to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3);
+inline
+DWORD   *pb_push4to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3, DWORD param4);
+
+inline
+void    pb_push(DWORD *p, DWORD command, DWORD nparam);
+
+inline
+DWORD   *pb_push1(DWORD *p, DWORD command, DWORD param1); //slow functions but with debug messages (targets SUBCH_3D (0))
```

Comment:
I'd remove the comment about this being slow. It's no longer true, and it always used to be very vague. These functions were never slow, unless debugging was enabled.

Reply:
Done. :+1: